ledger-core/v2/logicrunner/preprocessor/templates/wrapper.go.tpl

```diff
@@ -263,7 +263,10 @@ func Initialize() insolar.ContractWrapper {
 		GetPrototype: INSMETHOD_GetPrototype,
 		Methods: insolar.ContractMethods{
 			{{ range $method := .Methods -}}
-					"{{ $method.Name }}": INSMETHOD_{{ $method.Name }},
+					"{{ $method.Name }}": insolar.ContractMethod{
+					    Func: INSMETHOD_{{ $method.Name }},
+					    Unordered: {{ $method.Immutable }},
```

Comment:
It seems kinda strange. Do we now have 2 different types of call - unordered AND immutable? As I know, now we replace what we used to call 'immutable' with 'Unordered'. So I think we need to remove 'Immutable' from code base. And it is strange that we change the way wrapper is generated.

Reply:
If it's need to be done - it should be different PR

Reply:
https://insolar.atlassian.net/browse/PLAT-296